doc.html

```diff
@@ -10,7 +10,7 @@
 <body>
 
 <div class="header">
-Experimental ATLAS Service Template Specification
+ATLAS Service Template Specification
```

Comment:
By definition, versions 0.0.x are experimental. We just make it more obvious by mentioning it in the title. Please leave it until we are ready to go for 0.1.x